redisinsight/api/src/constants/error-messages.ts

```diff
@@ -52,4 +52,5 @@ export default {
   REDIS_MODULE_IS_REQUIRED: (module: string) => `Required ${module} module is not loaded.`,
   APP_SETTINGS_NOT_FOUND: () => 'Could not find application settings.',
   SERVER_INFO_NOT_FOUND: () => 'Could not find server info.',
+  INCREASE_MINIMUM_LIMIT: (count: string) => `Use a minimum of ${count} as the MAXSEARCHRESULTS.`,
```

Comment:
Maybe I'm wrong, but I don't agree with such error message. What does it represents? What we are trying to "avoid" by implementing this entire feature?

Reply:
Viktor sent a new message.